Casks/adoptopenjdk8-openj9.rb

```diff
@@ -1,68 +1,16 @@
 cask 'adoptopenjdk8-openj9' do
   version '8,202:b08'
-  sha256 '9ed76d2e4885bc1c7d9c937abaaf3739bddd2cd7c6deca5a46b79e94d62d421e'
+  sha256 :no_check
```

Comment:
Add a comment as to why no sha256 check in that case

Reply:
yeah can add Sha256 if we are happy with everything else?